e2e/features/ftpa-appellant-leadership-judge-decision-and-reasons.feature

```diff
@@ -0,0 +1,404 @@
+    And within the `FTPA Appellant Decision and Reasons documents` collection's first item, I should see `{$TODAY|D MMM YYYY}` for the `Date uploaded` field
+
+    When I switch to be a `Admin Officer`
+    And I click the `FTPA` tab
+    And I should see the `FTPA` page
+    And I should see `Permission refused` for the `The outcome of the application` field
+    And within the `FTPA Decision and Reasons document` collection's first item, I should see `FTPADecisionAndReasons.pdf` in the `Document` field
+    And within the `FTPA Decision and Reasons document` collection's first item, I should see `This is the ftpa decision and reasons` in the `Describe the document` field
+    And I should see `{$TODAY|D MMM YYYY}` in the `Decision date` field
+
+    Then I click the `Documents` tab
+    And within the `FTPA Appellant Decision and Reasons documents` collection's first item, I should see `FTPADecisionAndReasons.pdf` in the `Document` field
+    And within the `FTPA Appellant Decision and Reasons documents` collection's first item, I should see `This is the ftpa decision and reasons` in the `Description` field
+    And within the `FTPA Appellant Decision and Reasons documents` collection's first item, I should see `{$TODAY|D MMM YYYY}` for the `Date uploaded` field
+
+    When I switch to be a `Legal Rep`
+    And I click the `FTPA` tab
+    And I should see the `FTPA` page
+    And I should see `Permission refused` for the `The outcome of the application` field
+    And within the `FTPA Decision and Reasons document` collection's first item, I should see `FTPADecisionAndReasons.pdf` in the `Document` field
+    And within the `FTPA Decision and Reasons document` collection's first item, I should see `This is the ftpa decision and reasons` in the `Describe the document` field
+    And I should see `{$TODAY|D MMM YYYY}` in the `Decision date` field
+
+    Then I click the `Documents` tab
+    And within the `FTPA Appellant Decision and Reasons documents` collection's first item, I should see `FTPADecisionAndReasons.pdf` in the `Document` field
+    And within the `FTPA Appellant Decision and Reasons documents` collection's first item, I should see `This is the ftpa decision and reasons` in the `Description` field
+    And within the `FTPA Appellant Decision and Reasons documents` collection's first item, I should see `{$TODAY|D MMM YYYY}` for the `Date uploaded` field
+
+    When I switch to be a `Home Office POU`
+    And I click the `FTPA` tab
+    And I should see the `FTPA` page
+    And I should not see the `The outcome of the application` field
+    And I should not see the `FTPA Decision and Reasons document` field
+    And I should not see the `Decision date` field
+
+    Then I click the `Documents` tab
+    And I should not see `FTPA Appellant Decision and Reasons documents` field
```

Comment:
'the' is missing in this step and would fail as undefined step. Could you please check that.